.github/workflows/web.yml

```diff
@@ -397,7 +397,9 @@ jobs:
         e2e-test,
         performances-test,
       ]
-    if: ${{ contains(fromJson('["stable", "testing", "unstable", "pkgtest"]'), needs.get-version.outputs.stability) }}
+    if: ${{ !cancelled() && contains(fromJson('["stable", "testing", "unstable", "pkgtest"]'), needs.get-version.outputs.stability) && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') }}
```

Comment:
```suggestion
    if: ${{ contains(fromJson('["stable", "testing", "unstable", "pkgtest"]'), needs.get-version.outputs.stability) }}
```

Reply:
Thx, i was not sure about this change. I'll fix it.
Do i have to change that on every branches ?